core/btdiscovery.cpp

```diff
@@ -229,6 +229,11 @@ void BTDiscovery::getBluetoothDevices()
 			result.name = dev.callObjectMethod("getName", "()Ljava/lang/String;").toString();
 			qDebug() << "paired Device type" << btType << "with address" << result.address;
 			btPairedDevices.append(result);
+			if (btType == 3) { // DEVICE_TYPE_DUAL
```

Comment:
i think the separate `btType == 3` branch is redundant.

this section can be changed like so:
```
result.address = dev.callObjectMethod("getAddress","()Ljava/lang/String;").toString();
if (btType == 2 || btType == 3) // DEVICE_TYPE_LE or DEVICE_TYPE_DUAL
	result.address = QString("LE:%1").arg(result.address);
result.name = dev.callObjectMethod("getName", "()Ljava/lang/String;").toString();
qDebug() << "paired Device type" << btType << "with address" << result.address;
btPairedDevices.append(result);
```